jupyter_console/ptshell.py

```diff
@@ -115,7 +115,7 @@ def get_completions(self, document, complete_event):
         )
         start_pos = content['cursor_start'] - document.cursor_position
         for m in content['matches']:
-            yield Completion(m, start_position=start_pos)
+            yield Completion(m, start_position=start_pos, display_meta=content["metadata"].get(m, ""))
```

Comment:
Is this ok for us to use in jupyter_console @Carreau?

Reply:
I think it's unstable, but yes, you could use it when the kernel is IPython.